### PR TITLE
Fix preview or create tour logic

### PR DESCRIPTION
**Pull request by pandafulmanda**
screen shot 2017-12-04 at 6 41 14 pm

Once any course has been previewed:

screen shot 2017-12-04 at 5 02 22 pm

### CODE REVIEW

File: tutor/src/components/my-courses/index.jsx [hunk 1]
```diff
@@ -1,23 +1,56 @@
...
+    if (!this.props.tourContext.tour) {
+      const { stubbornTour } = this;
+
+      if (stubbornTour) {
+        stubbornTour.isEnabled = true;
+      }
+    }
+  }
+
   @computed get firstCourse() {
```
Comment (nathanstitt) [inline tutor/src/components/my-courses/index.jsx]: Hm, what're you trying to do here?  It seems like there's a tour that's refusing to display?

File: tutor/src/components/navbar/preview-add-course-btn.jsx [hunk 2]
```diff
@@ -26,6 +26,12 @@ export default class PreviewAddCourseBtn extends React.PureComponent {
     tourContext: React.PropTypes.instanceOf(TourContext),
   }
 
+  constructor(props) {
+    super(props);
+
+    this.onAddCourse.bind(this);
+  }
+
   @computed get course() {
```
Comment (nathanstitt) [inline tutor/src/components/navbar/preview-add-course-btn.jsx]: This shouldn't be needed, `onAddCourse` is decorated with `@action.bound`

Comment (pandafulmanda) [reply tutor/src/components/navbar/preview-add-course-btn.jsx]: i tried, and it errored without it, will check again

Comment (pandafulmanda) [reply tutor/src/components/navbar/preview-add-course-btn.jsx]: you're right,weird, i thought i saw it erroring, but it works :)

File: tutor/src/components/navbar/preview-add-course-btn.jsx [hunk 3]
```diff
@@ -46,7 +46,7 @@ export default class PreviewAddCourseBtn extends React.PureComponent {
   }
 
   render() {
-    if (!get(this, 'ux.showCreateCourseAction')) { return null; }
+    // if (!get(this, 'ux.showCreateCourseAction')) { return null; }
```
Comment (nathanstitt) [inline tutor/src/components/navbar/preview-add-course-btn.jsx]: 😁

**Pull request merged**